cmd/zfs_volume_wait/zfs_volume_wait

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+
+count_zvols() {
+	if [ -z "$zvols" ]; then
+		echo 0
+	else
+		echo "$zvols" | wc -l
+	fi
+}
+
+filter_out_zvols_with_links() {
+	while read -r zvol; do
+		if [ ! -e "/dev/$zvol" ]; then
```

Comment:
Should check `/dev/zvol/$zvol` instead of `/dev/$zvol`? According to the comment in `60-zvol.rules.in`, the `/dev/$zvol` link exists only for compatibility, and I'm guessing it could be missing if there is a collision between a pool name and some other file in `/dev/`

Reply:
You may also want to use `-L` instead of `-e` to ensure if it's a link incase someone drops a file in this directory.

Reply:
done